src/main/java/io/vertx/core/http/impl/HttpUtils.java

```diff
@@ -478,4 +479,47 @@ static HttpVersion toNettyHttpVersion(io.vertx.core.http.HttpVersion version) {
       return io.vertx.core.http.HttpMethod.OTHER;
     }
   }
+
+  private static final AsciiString TIMEOUT_EQ = AsciiString.of("timeout=");
+
+  public static int parseKeepAliveHeaderTimeout(CharSequence value) {
```

Comment:
Meh, I trust you. There wasn't any code in Netty which we could have reused?

Reply:
no